README.md

```diff
@@ -2,57 +2,60 @@
 
 # Bundestag.io &nbsp; <a href="https://github.com/kriasoft/nodejs-api-starter/stargazers" target="_blank"><img src="https://img.shields.io/github/stars/bundestag/bundestag.io.svg?style=social&label=Star&maxAge=3600" height="20"/></a>  <a href="https://twitter.com/democracy_de" target="_blank"><img src="https://img.shields.io/twitter/follow/democracy_de.svg?style=social&label=Follow&maxAge=3600" height="20"/></a>  <a href="https://www.facebook.com/democracygermany/" target="_blank"><img src="https://github.com/demokratie-live/democracy-assets/blob/master/docu/facebook.png" height="20"/></a>  <a href="https://discord.gg/Pdu3ZEV" target="_blank"><img src="https://github.com/demokratie-live/democracy-assets/blob/master/docu/discord.png" height="20"/></a>
 
-[![Build Status](https://travis-ci.org/bundestag/bundestag.io.svg?branch=master)](https://travis-ci.org/bundestag/bundestag.io)
+This is a GraphQL API for data from [bundestag.de].
 
-The Serversoftware for <a href="https://bundestag.io">Bundestag.io</a>. This is an API Defintion and Server for Data from <a href="https://www.bundestag.de">bundestag.de</a>
 
-## Tech Stack
-
-* [Node.js][node], [Yarn][yarn], [JavaScript][js], [Babel][babel]
-
-[More Dependecies](https://github.com/bundestag/bundestag.io/network/dependencies)
-
-![Projekt Struktur](https://github.com/demokratie-live/democracy-assets/blob/master/docu/api_structure_bundestag.png)
-
-## Prerequisites
-
-* [Node.js][node]
-* [MongoDB][mongo]
+![Project structure](./assets/api_structure_bundestag.png)
 
 ## Getting started
 
-Clone the git repo & run the project
+Run the following:
 ```
 git clone git@github.com:bundestag/bundestag.io.git
 cd bundestag.io
-yarn install
 ```
 
-### Configure environment variables
+### Without docker
+
+You need to have a [MongoDB][mongo] running as well as [NodeJS][node] installed
+on your system.
+
+If you need further configuration, you can do:
 ```
 cp .env.example .env
 ```
-Change .env if necessary.
+Change `.env` if necessary.
 
-### Compile and start
+Once this is set up, run the following:
 ```
-yarn ~~start~~ dev
+yarn install
+yarn run dev
 ```
+Now visit [localhost:3100][localhost] and you should see the GraphQL playground.
 
-### Diff scraped Data Versions
-```
-yarn diff
-```
+### Configuration
 
-### Test Project
+### With docker
+
+Run:
 ```
-yarn lint
+docker-compose up
```

Comment:
It looks like docker changed the command to use `docker compose`
```
Docker Compose is now in the Docker CLI, try `docker compose up`
```

Reply:
I'm on Archlinux and `docker --version` `20.10.6`. When I `docker compose up` without the dash I get:
```
docker: 'compose' is not a docker command.
See 'docker --help'
```

So maybe this is a Mac OS specific subcommand?

Reply:
yes maybe. both works for me, so lets let it be like it is .)